packages/neuron-ui/src/utils/validators.ts

```diff
@@ -13,7 +13,7 @@ export const verifyWalletSubmission = ({
   confirmPassword: string
   name: string
 }) => {
-  return password && name && password === confirmPassword
+  return password && name && password === confirmPassword && password.length >= 8 && confirmPassword.length >= 8
```

Comment:
no need for `confirmPassword.length >= 8` after check `password === confirmPassword`
```suggestion
  return password && name && password === confirmPassword && password.length >= 8
```

Reply:
ok